rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -763,7 +763,8 @@ error_code cellGameBootCheck(vm::ptr<u32> type, vm::ptr<u32> attributes, vm::ptr
 
 	if (_type == u32{CELL_GAME_GAMETYPE_HDD} && dirName)
 	{
-		strcpy_trunc(*dirName, Emu.GetTitleID());
+		ensure(dir.size() < CELL_GAME_DIRNAME_SIZE);
```

Comment:
We may need to demand it when booting games or going through HDD0/game directories when enumerating for game list.

Reply:
There is no evidence that any game even triggers this ensure.
I refuse to spend even a second on this.
"if it ain't broke, don't fix it"